### ISSUE TITLE
Unable to delete application tokens, or login with them

### ISSUE THREAD

**Issue by tomwganem** [SUSE/Portus#1975]
### Description

I was testing out 2.3.5, and then switch to HEAD because I wanted the option to prevent users from pushing to their own namespaces. Everything seems better, but applications tokens don't work. I can create them, but I can't delete them via the UI. Clicking the trash button next to the token doesn't seem to make any api call. ~~What's more, I am unable to login with any application token I create.~~

Edit: Couldn't login because I was doing it wrong. I've crossed out the steps that were meant to test that.

### Steps to reproduce

1. Checked out repo

2. `MACHINE_FQDN=portus.test.lan docker-compose up -d` (update /etc/hosts so that works)

3. Go `http://portus.test.lan:3000/`, Create a user and login

4. Add the registry using hostname `registry:5000`

~~5. test logging in to the registry from the cli~~

~~6. test logging in with a bad username/password~~

7. Go to http://portus.test.lan::3000/users/edit

8. Create application token with name "bar"

~~9. test logging in with the application token~~

10: Now try deleting the token by clicking the trash symbol

- **Expected behavior**: ~~I expect to be able to login with the application token.~~ I also expect to be able to delete application token.
- **Actual behavior**: ~~I can not login with application tokens I make.~~ I am unable to delete application tokens.

Trying to delete the token via the UI produces no rails logs

### Deployment information

Using docker-compose on Docker For Mac

```
$ docker version
Client:
 Version:           18.06.1-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        e68fc7a
 Built:             Tue Aug 21 17:21:31 2018
 OS/Arch:           darwin/amd64
 Experimental:      false

Server:
 Engine:
  Version:          18.06.1-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       e68fc7a
  Built:            Tue Aug 21 17:29:02 2018
  OS/Arch:          linux/amd64
  Experimental:     true
```
```
$ docker-compose version
docker-compose version 1.22.0, build f46880f
docker-py version: 3.4.1
CPython version: 3.6.4
OpenSSL version: OpenSSL 1.0.2o  27 Mar 2018
```

This is my docker-compose command
`MACHINE_FQDN=portus.test.lan docker-compose up -d`

**Configuration**:

```shell
$ git rev-parse HEAD^
0e1086bda5014d3d66f0229f897ec53c8ac56d5c
```
No custom configuration used.

**Comment by tomwganem** [comment]
Also, deployed opensuse/portus:head to kubernetes cluster with the same result

**Comment by vitoravelino** [comment]
> With that said, I'm still unable to delete tokens from the ui

Any error on the screen when you click on the trash button? Any error on devtools/logs? Is the user an admin or a normal user? I've tested both cases and it works for me (tm).

**Comment by vitoravelino** [comment]
I didn't try with `2.3.5` but `2.4` and `head` image are working for me. Without a precised error of what's happening (e.g. devtools log), I'm unable to help with any feedback.

**Comment by ldubrois** [comment]
I am running in the same issue, I cannot remove Application Tokens.
I have created a user tagged as bot. I have created 2 tokens named T1 and T2 (to make easier to understand).
I wanted to remove both of the tokens. I successfully removed T2, but I did not succeeded with T1.
I thought at least one token should be present for a bot user, so I create a T3. And now I cannot delete T1 or T3.

I am running Portus 2.4.1@ddc403833c8975a83aa93d01baa4bc10acd40c1b in a swarm stack.
When I click the Yes button, nothing happens, neither in JS console, Browser network inspector, container's log.
